package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "start": "node .",
     "pretest": "jshint .",
-    "posttest": "nsp check"
+    "posttest": "nsp check",
+    "api": "node .",
+    "client": "./node_modules/.bin/webpack-dev-server --config server/webpack.dev.config.js"
```

Comment:
I just learned that npm allows you to just specify `webpack-dev-server` sans the `./node_modules/.bin/` bit npm will first check in your local `node_modules`